packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -366,12 +332,9 @@ export const makeInnerVault = (
     if (phase === VaultPhase.ACTIVE) {
       assertProposalShape(seat, {
         give: { RUN: null },
-        want: { Collateral: null },
       });
 
-      // you're paying off the debt, you get everything back. If you were
-      // underwater, we should have liquidated some collateral earlier: we
-      // missed our chance.
```

Comment:
I've found this clarification helpful before. principle for removal?

Reply:
It presumes that we should have liquidated you when we had the chance, which is not the point of liquidation. In this case, if the user can pay off the debt, the vault is solvent and the contract should be happy.